### PR TITLE
Add region flags to exomes all sites AN

### PR DESCRIPTION
**Pull request by jkgoodrich**
Job: 26b81963ebd84a9498a55bb4c46debc6

### CODE REVIEW

File: gnomad_qc/v4/annotations/compute_coverage.py [hunk 1]
```diff
@@ -515,12 +533,22 @@ def main(args):
...
-            logger.info("Exporting all sites AN tsv...")
-            res = coverage_resources.export_all_sites_an_release_tsv
+        if args.export_all_sites_an_release_files:
+            logger.info("Exporting all sites AN HT...")
+            res = coverage_resources.export_all_sites_an_release_files
             res.check_resource_existence()
-            ht = res.allele_number_ht.ht()
-            ht = ht.select(AN=ht.AN[0])
+
+            ht = res.an_qual_hist_ht.ht()
```
Comment (KoalaQin) [inline gnomad_qc/v4/annotations/compute_coverage.py:541]: One basic question, I was looking for input resources, it seems that you used AN_qual_hist HT to get a new AN HT with regions flags now, why did we output a AN HT before separately?

**Review by KoalaQin — COMMENTED** [review]
LGTM, I just double checked if the number of variants outside capture/calling region makes sense, I saw it's consistent trend as in the exome release HT, in the all sites AN HT, there are fewer variants outside Broad calling regions than outside UKB calling regions.

File: gnomad_qc/v4/annotations/compute_coverage.py (hunk 1 above)
Comment (jkgoodrich) [reply gnomad_qc/v4/annotations/compute_coverage.py:541]: I need the qual hists for the combined freq creation, but we only want to release the all sites AN, we don't need to release the qual hists

**Review by KoalaQin — APPROVED** [review]
LGTM!

**Pull request merged**